labop/execution_engine.py

```diff
@@ -348,34 +429,59 @@ def protocol_execution_aggregate_child_materials(self):
     ----------
     self: ProtocolExecution object
     """
-    child_materials = [e.call.consumed_material for e in self.executions
-                       if isinstance(e, labop.CallBehaviorExecution) and
-                          hasattr(e.call, "consumed_material")]
+    child_materials = [
+        e.call.consumed_material
+        for e in self.executions
+        if isinstance(e, labop.CallBehaviorExecution)
+        and hasattr(e.call, "consumed_material")
+    ]
     specifications = {m.specification for m in child_materials}
-    self.consumed_material = (labop.Material(s,sum_measures([m.amount for m in child_materials if m.specification==s]))
-                              for s in specifications)
-labop.ProtocolExecution.aggregate_child_materials = protocol_execution_aggregate_child_materials
-
-
-def protocol_execution_to_dot(self, execution_engine=None, ready: List[uml.ActivityNode] = [], done=set([])):
+    self.consumed_material = (
+        labop.Material(
+            s,
+            sum_measures(
+                [m.amount for m in child_materials if m.specification == s]
+            ),
+        )
+        for s in specifications
+    )
+
+
+labop.ProtocolExecution.aggregate_child_materials = (
+    protocol_execution_aggregate_child_materials
+)
+
+
+def protocol_execution_to_dot(
+    self,
+    execution_engine=None,
+    ready: List[uml.ActivityNode] = [],
+    done=set([]),
+    out_dir="out"
+):
     """
     Create a dot graph that illustrates edge values appearing the execution of the protocol.
     :param self:
     :return: graphviz.Digraph
     """
-    dot = graphviz.Digraph(comment=self.protocol,
-                           strict=True,
-                           graph_attr={"rankdir": "TB",
-                                       "concentrate": "true"},
-                           node_attr={"ordering": "out"})
+    dot = graphviz.Digraph(
+        comment=self.protocol,
+        strict=True,
+        graph_attr={"rankdir": "TB", "concentrate": "true"},
+        node_attr={"ordering": "out"},
+    )
+
     def _make_object_edge(dot, incoming_flow, target, dest_parameter=None):
         flow_source = incoming_flow.lookup().token_source.lookup()
         source = incoming_flow.lookup().edge.lookup().source.lookup()
         value = incoming_flow.lookup().value.get_value()
-        #value = value.value.lookup() if isinstance(value, uml.LiteralReference) else value.value
+        is_ref = isinstance(incoming_flow.lookup().value, uml.LiteralReference)
+        # value = value.value.lookup() if isinstance(value, uml.LiteralReference) else value.value
 
         if isinstance(source, uml.Pin):
-            src_parameter = source.get_parent().pin_parameter(source.name).property_value
+            src_parameter = (
```

Comment:
Are the parentheses here for line continuation recommended by PEP 8 style guide, or is it a personal preference?  I don't have a problem with it, just trying to get a feel for your stylistic preferences.